app/models/create_like.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLike < ApplicationRecord
+  belongs_to :author, class_name: 'CreateUser', foreign_key: :author_id
+  belongs_to :create_posts, class_name: 'CreatePost', foreign_key: :create_posts_id
+  after_save :update_likes_counter
+
+  def update_likes_counter
+    post.increment(:likes_counter)
+  end
```

Comment:
- [ ] [OPTIONAL] Just like I have recommended in the Post and Comment counter. We can have the same for the Like counter as well:
`after_save :update_likes_counter`